projects/ad9361/src/main.c

```diff
@@ -421,7 +450,13 @@ int main(void)
 #ifdef LINUX_PLATFORM
 	gpio_init(default_init_param.gpio_sync);
 #endif
-	gpio_direction(default_init_param.gpio_sync, 1);
+	status = gpio_get(&default_init_param.gpio_desc_sync,
+			  default_init_param.gpio_sync);
+	if (status < 0) {
```

Comment:
This is probably fine, but above you are checking the status against SUCCESS: `if (status != SUCCESS)`. For consistency purposes, I would stick to only one approach.